packages/proxy/lib/http/util/prerequests.ts

```diff
@@ -82,9 +87,10 @@ export class PreRequests {
   pendingPreRequests = new QueueMap<PendingPreRequest>()
   pendingRequests = new QueueMap<PendingRequest>()
   sweepIntervalTimer: NodeJS.Timeout
+  protocolManager?: ProtocolManagerShape
 
   constructor (
-    requestTimeout = 500,
+    requestTimeout = 1000 * 2,
```

Comment:
why the `* 2` when you could just set it to 2000?

Reply:
I was trying to match the style of the next line. You're right though it's simple math. Could just update